config/ignition_fortress_ubuntu_jammy.yaml

```diff
@@ -68,12 +68,12 @@ filter_formula: "\
  Package (= libignition-physics5-tpe-dev) |\
  Package (= libignition-physics5-tpelib) |\
  Package (= libignition-physics5-tpelib-dev) \
-), ($Version (% 5.1.0+osrf-2*) | $Version (% 5.1.0+osrf-3*)) |\
+), ($Version (% 5.2.0-1*) | $Version (% 5.1.0+osrf-3*)) |\
 (Package (= ignition-plugin) |\
  Package (= libignition-plugin) |\
  Package (= libignition-plugin-dbg) |\
  Package (= libignition-plugin-dev) \
-), $Version (% 1.2.1+osrf-1*) |\
+), $Version (% 1.3.1-1*) |\
```

Comment:
I only see 1.3.0-1: http://packages.osrfoundation.org/gazebo/ubuntu-stable/pool/main/i/ignition-plugin/

```suggestion
), $Version (% 1.3.0-1*) |\
```

![image](https://user-images.githubusercontent.com/5751272/184967350-a757aac7-3f87-4b10-8d6c-a16ed6655c73.png)

Reply:
dah, I was using buggy one-liners instead of the dashboard. Sorry.